src/ContextProvider/index.js

```diff
@@ -5,28 +5,28 @@ import { Component, PropTypes, Children } from "react"
 export default class StatinamicContext extends Component {
   props: {
     collection: StatinamicCollection,
-    layouts: Object,
+    layouts: Object, // deprecated
```

Comment:
I know this is not the scope of this PR but flowtype props and propTypes is not the same here.

collection, layouts and metadata is required in flowtype.